src/configmanager.h

```diff
@@ -103,7 +103,7 @@ class ConfigManager
 			MAX_MARKET_OFFERS_AT_A_TIME_PER_PLAYER,
 			EXP_FROM_PLAYERS_LEVEL_RANGE,
 			MAX_PACKETS_PER_SECOND,
-
+			LIVE_CAST_PORT,
```

Comment:
This aswell as ENABLE_LIVE_CASTING on line 47 should be moved up, so there is a new line to the LAST_*